build/buildBuildImageBases.sh

```diff
@@ -52,13 +52,13 @@ case $IMAGE_DIR_TO_BUILD in
 		docker build -f $BUILD_IMAGES_DIR/python/prereqs/Dockerfile -t "python-build-prereqs" $REPO_DIR
 		buildImages "python"
 		;;
-	'php')
-		echo "Building PHP base images"
-		echo
-
-		docker build -f $BUILD_IMAGES_DIR/php/prereqs/Dockerfile -t "php-build-prereqs" $REPO_DIR
-		buildImages "php"
-		;;            
+	#'php')
```

Comment:
Can you send out a separate clean up PR?

Reply:
ok .. I will remove from here